src/encoded/types/file.py

```diff
@@ -66,6 +66,14 @@ def property_closure(request, propname, root_uuid):
     return seen
 
 
+def is_md5sum_constrained_file(file_properties, unique_md5_output_categories, uniform_processing_pipeline_uuid):
+    if ((file_properties.get('lab') != uniform_processing_pipeline_uuid or 
```

Comment:
This could still be cleaned up a bit:

```
def is_md5sum_constrained():
    conditions = [
        properties.get('lab') != ENCODE_PROCESSING_PIPELINE_UUID,
        properties.get('output_category') == RAW_DATA
    ]
    if any(conditions):
        return True
    return False
```

Reply:
thanks, done